src/transformers/generation/configuration_utils.py

```diff
@@ -485,8 +485,11 @@ def __init__(self, **kwargs):
         # Validate the values of the attributes
         self.validate(is_init=True)
 
+        # Finally, store the original hash of the object (to detect post-loading changes)
```

Comment:
(the changes in `GenerationConfig` were made to align logic with `PreTrainedConfig`)